### ISSUE TITLE
Dont use Overwrite mixins

### ISSUE THREAD

**Issue by MehVahdJukaar** [IzzelAliz/Arclight#1558]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

1.20.1-1.0.6-SNAPSHOT

### OS & Java versions

21

### Plugins and Mods

```raw
no plugins
```

### Description

The mod uses some @Overwrite mixins.
These are fundamentally incompatible with other mods that might inject in those methods and should never be used.

An example is the creeper entity mixin

**Comment by qyl27** [comment]
Yes, we have now replaced `@Overwrite` with `@Decorate` in 1.20.4+.
